equinix/resource_metal_vrf.go

```diff
@@ -1,26 +1,20 @@
 package equinix
 
 import (
-	"log"
-	"time"
-
+	"context"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/packethost/packngo"
+	"log"
 )
 
 func resourceMetalVRF() *schema.Resource {
 	return &schema.Resource{
-		Timeouts: &schema.ResourceTimeout{
```

Comment:
This is removing timeouts entirely rather than fixing them.  I don't necessarily think that's an issue:  as I understand it, they didn't work and they weren't documented, so IMO it's just as valid to remove them as it is to make them work.  However, if we're removing them, then the PR title should be updated to reflect that.

Reply:
Yes thats exactly correct, added the description in PR description as well